### PR TITLE
ZEN-3514 KZOE: Change SwagEdit to KaiZen OpenAPI Editor in UI

### PR DESCRIPTION
**Pull request by tfesenko**
## Transparent background for icons
screen shot 2017-05-08 at 2 22 04 pm
screen shot 2017-05-08 at 2 22 12 pm

## Updated labels for the New Wizard category:
screen shot 2017-05-08 at 2 22 17 pm

## New root for preferences which aggregates prefs for Swagger v2 and OpenAPI v3:
screen shot 2017-05-08 at 2 22 27 pm

## Plugin names
screen shot 2017-05-08 at 2 25 33 pm

### CODE REVIEW

**Comment by tedepstein** [comment]
Corrected the summary here, and in the Jira issue.  "Editor", not "Parser".

**Comment by tedepstein** [comment]
@tfesenko , we use hyphens in the github project name, because we can't use spaces.  But in human-readable naming, we shouldn't use hyphens.  So it should be KaiZen OpenAPI Editor, not KaiZen-OpenAPI-Editor.

**Comment by tedepstein** [comment]
File New Wizard:  The word "Spec" should have a capital 'S', same as with Swagger 2.0 Spec.

**Comment by tedepstein** [comment]
@tfesenko , here's a complete review of the screenshots. 

File > New Dialog
* Category should be "KaiZen OpenAPI Editor" (no hyphens)
* Items should be named consistently.
   * First option should be "Swagger v2 Spec" (Should not include the word "New" in this context, and should include "v2" for clarity + consistency.)
   * Second option should be "OpenAPI v3 Spec" (with upper case 'S' in 'Spec'). 

Preferences Dialog:
* Category can be just "KaiZen" 
* Description should be "KaiZen OpenAPI Editor Preferences" (without hyphens). 
* Sub-category should be "OpenAPI v3"  (No space between "Open" and "API")

Plug-In Names:
* Remove Hyphens

Feature Name: Not included in this PR yet, but we do eventually want it renamed.  If it's going to be disruptive to the API Studio build, it can wait.

**Comment by tedepstein** [comment]
@tfesenko , I don't see a KZOE "Tile" in the about dialog.  We don't need one right now, but if it's easy (trivial) to add, here's an image for it.  

kaizen symbol v2

I will add to Jira as a lower priority task.

**Comment by tfesenko** [comment]
@tedepstein renamed GUI items as you suggested:
screen shot 2017-05-08 at 3 41 23 pm
screen shot 2017-05-08 at 3 42 08 pm


Plugin and feature names are on the way...

**Comment by tedepstein** [comment]
Looks good, @tfesenko .  Taking a quick look at code changes...

**Comment by tfesenko** [comment]
Updated plugin and feature names (feature name is not visible in GUI in my config):
screen shot 2017-05-08 at 3 58 47 pm

File: com.reprezen.swagedit.feature/feature.xml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <feature
       id="com.reprezen.swagedit.feature"
-      label="SwagEdit"
+      label=" KaiZen-OpenAPI-Editor"
```
Comment (tedepstein) [inline com.reprezen.swagedit.feature/feature.xml]: Does this label need to be machine-readable, without whitespace?  If it's a label for humans, please replace hyphens with spaces.

Comment (tfesenko) [reply com.reprezen.swagedit.feature/feature.xml]: @tedepstein fixed in https://github.com/RepreZen/KaiZen-OpenAPI-Editor/pull/279/commits/97a54ba50e3a98f6e1e87c5ae542218faf6bb3a6#diff-abd602282327c9fa1b93b03213da3def

Comment (tedepstein) [reply com.reprezen.swagedit.feature/feature.xml]: BTW, the label here appears to have a leading space.  Is that intentional?

Comment (tfesenko) [reply com.reprezen.swagedit.feature/feature.xml]: Leading space removed in 	c1a8646

**Comment by tedepstein** [comment]
OK, merging.

**Pull request merged**